sdk/src/main/java/org/coralibre/android/sdk/internal/database/persistent/DaoExposureInformation.java

```diff
@@ -0,0 +1,25 @@
+package org.coralibre.android.sdk.internal.database.persistent;
+
+import androidx.room.Dao;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+
+import org.coralibre.android.sdk.internal.database.persistent.entity.EntityExposureInformation;
+
+import java.util.List;
+
+@Dao
+public interface DaoExposureInformation {
+
+    @Insert(onConflict = OnConflictStrategy.ABORT)
+    public void insertExposureInformations(List<EntityExposureInformation> entityExposureInformations);
```

Comment:
I don't think "informations" is a valid word in English, but nevermind, ignore this ;-)